python/tank/descriptor/io_descriptor/appstore.py

```diff
@@ -104,6 +104,7 @@ class IODescriptorAppStore(IODescriptorBase):
         "tag_list",
         "sg_detailed_release_notes",
         "sg_documentation",
+        "sg_branch",
```

Comment:
It is needed to cache the value, which at the moment is only for reference purpose. I thought it was better to retrieve and cache the value, even if we don't do anything with it right now. Later we could want to display it or use it to implement a dedicated behaviour when using `label` with a branch release. I can remove it if it seems overkill?